<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HA1101901716" name="SM-7.0.0.0.700007-e55-01.ova" tests="3" failures="1" timestamp="28 Sep 2015 08:25:06 GMT" time="302.451" errors="0">
  <testcase name="AddVM" time="37.07" classname="com.avaya.sdmclient.runnerdemo.sdmSMGRConcurrent">
    <failure type="com.avaya.sdmclient.runnerdemo.MyException" message="This VM Name already exists! Please try another name.
Check Screenshot for the same.
">
      <![CDATA[com.avaya.sdmclient.runnerdemo.MyException: This VM Name already exists! Please try another name.
Check Screenshot for the same.

	at com.avaya.sdmclient.Settings.errorBox(Settings.java:419)
	at com.avaya.sdmclient.runnerdemo.sdmSMGRConcurrent.AddVM(sdmSMGRConcurrent.java:260)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:648)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:834)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1142)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- AddVM -->
  <testcase name="startNewThread" time="265.337" classname="com.avaya.sdmclient.runnerdemo.sdmSMGRConcurrent"/>
  <testcase name="DeleteVM" time="0.0" classname="com.avaya.sdmclient.runnerdemo.sdmSMGRConcurrent">
    <skipped/>
  </testcase> <!-- DeleteVM -->
</testsuite> <!-- SM-7.0.0.0.700007-e55-01.ova -->
